platform-sdk/consensus-otter-tests/build.gradle.kts

```diff
@@ -59,8 +94,8 @@ tasks.compileTestFixturesJava {
 // Runs tests against the Turtle environment
 tasks.register<Test>("testTurtle") {
```

Comment:
This could be moved into the test suite as additional target. You then do not need to set the details anymore.

For that, remove this whole `task.register`. And instead do this inside the `targets {` block above.

```
                register("testTurtle") {
                    testTask {
                        systemProperty("otter.env", "turtle")
                        // other system properties are configured by the `all {` block for all tasks
                    }
                }
```

I would prefer that to have less custom configuration.